modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,537 @@
+
+
+=== Parsing large documents no longer throws a Maximum call stack size exceeded exception
+
+Previously, parsing very large documents would throw an error to the console that they had exceeded the Maximum call stack size. 
+
+For example:
+
+[listing]
+----
+tinymce.min.js:9 Uncaught RangeError: Maximum call stack size exceeded
+    at tinymce.min.js:9:144118
+    at zp (tinymce.min.js:9:143461)
+    at zp (tinymce.min.js:9:143480)
+    at zp (tinymce.min.js:9:143507)
+    at zp (tinymce.min.js:9:143507)
+----
+
+the {productname} 6.2 parsing logic was updated and {productname} can now handle large documents without throwing this error.
+
+=== DomParser filter matching was not checked between filters, which could lead to an exception in the parser
+
+When inserting content, each node has their attributes run through a number of filters.
+
+By default, only the filters within {productname} are run. Users are, however, able to add their own filters to run before any {productname} filters, using the xref:apis/tinymce.html.domparser.adoc[DomParser API].
+
+Previously, this node filter function did not check whether the next filter was still valid for all nodes. This meant the {productname} filters naively assumed that each attribute would be available for each filter.
+
+This situation, which was not identified by default, could, when using a custom filter, potentially remove a necessary attribute for the next filter. This resulted in a console error.
+
+This has been fixed in {productname} 6.2 by checking that each filter is still valid with the current attributes before running that filter.
+
+
+=== Fixed `contenteditable="false"` lists and `contenteditable="true"` list elements issues
+
+Previously, lists with `contenteditable="false"` attributes could be toggled and updated. And, within those lists, list elements with `contenteditable="true"` attributes could be indented, split into another list element, or appended to the previous list element by deletion.
+
+Neither behavior was correct and, as of {productname} 6.2, this has been corrected.
+
+In {productname} 6.2, list functionality in this regard is dependent on the state of the closest inheritable `contenteditable` attribute.
+
+If a `list` has a `contenteditable` attribute value of its own, this is, of course the closest ‘inheritable’ attribute, and the list and its child elements take that value.
+
+If a `list` does not have a `contenteditable` attribute value of its own and the immediate parent of the `list` is a `div` which does have a `contenteditable` attribute set, the `list` will inherit the value of this attribute.
+
+However, if the `list` has a `contenteditable` attribute value of its own it will take this value rather than the value of the parent div.
+
+Likeiwse, if a child element of a `list` (for example, an `+<li>+`) has a `contenteditable` attribute value of its own, it will take this value rather than the value of any parent element.
```

Comment:
```suggestion
Likewise, if a child element of a `list` (for example, an `+<li>+`) has a `contenteditable` attribute value of its own, it will take this value rather than the value of any parent element.
```

Reply:
(discussion): accept suggestion